src/test/resources/logback-test.xml

```diff
@@ -20,7 +20,7 @@
 
   <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
     <filter class="ch.qos.logback.classic.filter.ThresholdFilter">
-      <level>INFO</level>
+      <level>DEBUG</level>
```

Comment:
I think INFO is fine, the tests already print way too much output

Reply:
I don't have a strong opinion on this, can revert since these are tests.

@yifan-c - Does this make sense, since you recommended this change?

Reply:
yeah, let's keep info for now until we make logging more significant for tests